pkg/driver/node_linux.go

```diff
@@ -101,7 +101,7 @@ func (d *nodeService) findDevicePath(devicePath, volumeID, partition string) (st
 
 	if util.IsSBE(d.metadata.GetRegion()) {
 		klog.V(5).InfoS("[Debug] Falling back to snow volume lookup", "devicePath", devicePath)
-		canonicalDevicePath = "/dev/vd" + strings.TrimPrefix(devicePath, "/dev/xvdb")
+		canonicalDevicePath = "/dev/vd" + devicePath[len(devicePath)-1:]
```

Comment:
np: Can you add a comment here explaining why you're doing this?

Reply:
For my own understanding: the 26-volume limit doesn't exist for the current implementation, correct?